R/methods-Megastudy.R

```diff
@@ -22,6 +22,26 @@ setMethod('getVariableSpec', signature('ANY'), function(object) {
   return(object@variableSpec)
 })
 
+# this might be unexpected behavior. should there be a param to choose between the collection and its member specs?
+#' @export
+setMethod('getVariableSpec', signature('VariableMetadata'), function(object, getCollectionMemberVarSpecs = c(TRUE, FALSE)) {
+  getCollectionMemberVarSpecs <- veupathUtils::matchArg(getCollectionMemberVarSpecs)
+  varSpecs <- list(object@variableSpec)
+
+  if (object@isCollection && getCollectionMemberVarSpecs) {
+    varSpecs <- as.list(object@members)
```

Comment:
do we need to also get the var specs of the members? I dont remember what the members slot looks like

Reply:
the members slot is a VariableSpecList, im converting that here to a standard list of VariableSpecs